.ci/pitest.sh

```diff
@@ -10,16 +10,6 @@ function checkPitestReport() {
   grep -irE "$SEARCH_REGEXP" target/pit-reports \
      | sed -E 's/.*\/([A-Za-z]+.java.html)/\1/' | LC_ALL=C sort > target/actual.txt
   printf "%s\n" "${ignored[@]}" | sed '/^$/d' | LC_ALL=C sort > target/ignored.txt
-  if [ -n "$2" ] ; then
-      local -n unstableMutations=$2
```

Comment:
Please restore functionality to support unstable.
Just not use unstable.

Reply:
Done.